java.html.markdown

```diff
@@ -225,7 +225,7 @@ public class LearnJava {
         String fooString = "My String Is Here!";
 
         // Text blocks
-        vat textBlock = """
+        var textBlock = """
```

Comment:
The previous section introduces `var`, which allows for type inference. To my knowledge, `vat` isn't valid Java:

```java
jshell> vat textBlock = "invalid";
|  Error:
|  cannot find symbol
|    symbol:   class vat
|  vat textBlock = "invalid";
|  ^-^
```